src/main/java/com/ncedu/cheetahtest/controller/userProfile/UserController.java

```diff
@@ -0,0 +1,50 @@
+package com.ncedu.cheetahtest.controller.userProfile;
+
+import com.ncedu.cheetahtest.entity.security.RegisterResponse;
+import com.ncedu.cheetahtest.entity.user.User;
+import com.ncedu.cheetahtest.service.user.UserService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api")
+@Slf4j
+@CrossOrigin(origins = "http://localhost:4200")
```

Comment:
It would be better to inject the URL of the frontend server from the properties file